webpack/webpack.config.cjs

```diff
@@ -175,6 +175,10 @@ module.exports = function(env, argv) {
         },
       },
     },
+    cache: {
+      type: 'filesystem',
+      allowCollectingMemory: true,
```

Comment:
```suggestion
```

From https://webpack.js.org/configuration/other-options/#cacheallowcollectingmemory it looks like this already defaults to true in development mode.  (Unless I'm misunderstanding?)